app/src/app/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,78 @@
+"use client"
+import {useEffect, useState} from 'react';
+import {useMapStore} from '../store/mapStore';
+import {AlertDialog, Button, Flex, Text} from '@radix-ui/themes';
+import * as Toast from '@radix-ui/react-toast';
+
+export const ErrorNotification = () => {
```

Comment:
PP: Good for now but there's probably an argument to be made either way that this should be an error dialog.

Reply:
Severity 1 is a dialog!